src/openseadragon.js

```diff
@@ -206,6 +206,9 @@
   * @property {Number} [opacity=1]
   *     Opacity of the drawer (1=opaque, 0=transparent)
   *
+  * @property {String|Function} [placeholderFillStyle=null]
```

Comment:
It's pretty powerful, but you can pick most of it up by looking at examples here.